src/routes/index.svelte

```diff
@@ -6,16 +6,21 @@
 	<div class="w-full pb-10">
 		<h1>Welcome to The small Society</h1>
 		<p>
-			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
-			labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
-			laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
-			voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
-			non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-			<br /> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
-			ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-			nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit
-			esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt
-			in culpa qui officia deserunt mollit anim id est laborum.
+			Pete Dennis is the founder of the SMALL society, the umbrella organisation for
```

Comment:
@LizG11 do we want to refer to TsS as `The small Society` or `The SMALL Society`?

Reply:
I’ll check with Pete and confirm

Reply:
Are we happy to address this within the catchall branch https://github.com/lhollyer/the-small-society/tree/pre-launch-tweaks?

Reply:
The website should read The small Society or abbreviation to TsS. Thanks for checking!

Reply:
Happy to address in that branch